src/DynamoCoreWpf/ViewModels/PackageManager/PublishPackageViewModel.cs

```diff
@@ -893,6 +893,7 @@ public bool RetainFolderStructureOverride
                 }
             }
         }
+        private string CurrentPackageDirectory { get; set; }
```

Comment:
can you add a comment on what this represents? Is it the root directory of the package? Or it's parent that contains all packages, or what?

Reply:
root directory of the package, will add a comment